vassal-app/src/main/java/VASSAL/build/module/GameState.java

```diff
@@ -661,6 +681,14 @@ public boolean loadGame(File f, boolean continuation) {
         loadContinuation(f);
       }
       else {
+        int optionToSave = NO_NEED_TO_SAVE;
```

Comment:
This is dangerous. What happens if the values of the `JOptionPane` constants are changed?

Reply:
I guess _theoretically_ but the chances of suddenly using -2 seem ... effectively zero? Compared to making a bunch of extra lines of code to translate from one scheme to another?